src/sonic-host-services/scripts/hostcfgd

```diff
@@ -6,10 +6,13 @@ import ipaddress
 import os
 import subprocess
 import syslog
+import signal
+import sys
```

Comment:
Seems like you change order during backport. It will make others backport difficult in general. Could you try minimize new changes during the backport?

Reply:
Reordered imports and added safe_eval method